include/alpaka/queue/sycl/Utility.hpp

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2021 Jan Stephan
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+
+#pragma once
+
+#ifdef ALPAKA_ACC_SYCL_ENABLED
+
+#    include <CL/sycl.hpp>
+
+#    include <algorithm>
+#    include <type_traits>
+#    include <vector>
+
+namespace alpaka::experimental::detail
+{
+    template<typename T, typename = void>
+    struct is_sycl_enqueueable : std::false_type
+    {
+    };
+
+    template<typename T>
+    struct is_sycl_enqueueable<T, std::void_t<decltype(T::is_sycl_enqueueable)>> : std::true_type
+    {
+    };
+
+    inline auto remove_completed(std::vector<sycl::event>& events)
```

Comment:
```suggestion
    inline auto remove_completed(std::vector<sycl::event>& events) -> void
```

Reply:
Why?

Reply:
Because I thought this function returned something until I read the entire body. I think it would help readability a bit. But no hard opinion.